index.html

```diff
@@ -7,57 +7,215 @@
+                        </div>
+                        <div class="image"></div>
+                        <div class="card-body">
+                            <h5>Multi-Post Stories</h5>
+                                <p>A daily selection of privately
+                                    personalized reads; no accounts or sign-ups required.
+                                    has been the industry's standard dummy text ever since
+                                    the 1500s, when an unknown printer took a standard dummy text.
+                            </p>
+                            <ul>
+                                <li class="css">CSS</li>
+                                <li class="html">HTML</li>
+                                <li class="bootstrap">Bootstrap</li>
+                                <li class="ruby">Ruby</li>
+                            </ul>
+                            <button type="button">See project</button>
+                        </div>
+                    </article>
+                    <!-- cards -->
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                       </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
```

Comment:
- [ ] Kindly ensure that this container aligns vertically with the paragraph text above it, to achieve a pixel-perfect design.

- Your design
<img width="145" alt="wrong" src="https://user-images.githubusercontent.com/41163571/227705975-deb6374c-8d08-49b6-a714-a8539d7d2102.png">

- Figma design
<img width="205" alt="right" src="https://user-images.githubusercontent.com/41163571/227705989-c86bd085-72be-4761-bc35-5ee5a79f1bc0.png">

Reply:
Fixed the alignment as instructed.